view/index.html

```diff
@@ -40,7 +40,7 @@ <h3 class="crunch-top"><span class="pull-left glyphicon glyphicon-exclamation-si
   {{#deleted}}
       <div class="col-md-12 clearfix" data-test="alert">
           <div class="alert alert-info">
-              <button aria-hidden="true" data-dismiss="alert" class="close" type="button">×</button>
```

Comment:
Hm. I wonder what the original justification was for applying `aria-hidden` to this. The "x" isn't a good accessible name, which might be why it was hidden to AT (which would have had the side-effect of making it harder to use for voice input users). 

Maybe the "x" should be changed to a much less ambiguous "close" instead. Do you have a screenshot of what this button looks like on the page?

Reply:
Yes, so earlier it was nearly invisible -
<img width="1159" alt="Screenshot 2021-05-26 at 09 35 29" src="https://user-images.githubusercontent.com/11158082/119621771-cdfe2c00-be06-11eb-9c2e-4adc0b5add2d.png">
Then with these changes it will look like -
<img width="1152" alt="Screenshot 2021-05-26 at 09 40 46" src="https://user-images.githubusercontent.com/11158082/119621859-e5d5b000-be06-11eb-9c05-c96c6f065ddb.png">

Reply:
oh yeah that looks much better. I'm going to make a code suggestion on the next line 😄

Reply:
... actually, no I'm not, because the button for making suggestions seems to have vanished 🤷‍♀️ 

I wanted to suggest:

```<button data-dismiss="alert" class="close" type="button" aria-label="close">×</button>```

in line with the [ARIA 14 technique](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA14) guidance.